optionals/compat_rhs_afrf3/CfgWeapons.hpp

```diff
@@ -263,8 +278,9 @@ class CfgWeapons {
         modes[] = {};
         picture = "\rhsafrf\addons\rhs_heavyweapons\data\ico\rhs_SPG9_INS_ca.paa";
     };
+
     class GVAR(spg9m_carry): GVAR(spg9_carry) {
-        class ACE_CSW {
+        class ACE_CSW: ACE_CSW {
```

Comment:
Why?

Reply:
It didn't inherit the deploy and pickup attributes, so i added inheritance. Can hardcode them there instead.

Reply:
Ah, ok, so this was a different bug. Seems weird to me though. However, this spg9m_carry class is also missing the author attribute, which cannot be inherited, yet it does inherit public scope. There seem to be more problems with this class.